.github/actions/occt-install-dep/action.yml

```diff
@@ -24,13 +24,16 @@ runs:
         repository: Open-Cascade-SAS/OCCT
         submodules: true
         path: './dependencies/occt'
-        ref: V7_6_3
+        ref: V7_7_2
 
-    - name: Patch OCCT INSTALL_RPATH
+    - name: Patch OCCT
       if: steps.cache-occt.outputs.cache-hit != 'true'
       working-directory: ${{github.workspace}}/dependencies/occt
       shell: bash
-      run: patch -p1 < $GITHUB_ACTION_PATH/0001-Adding-INSTALL_RPATH-Linux.patch
+      run: |
+        patch -p1 < $GITHUB_ACTION_PATH/0001-Adding-INSTALL_RPATH-Linux.patch
+        patch -p1 < $GITHUB_ACTION_PATH/occt-remove-unneeded-includes.patch
```

Comment:
Is there an issue opened upstream to inform about these issues?

Reply:
https://github.com/Open-Cascade-SAS/OCCT does not have issues open, to report an issue, you need to access https://tracker.dev.opencascade.org/my_view_page.php and I've never been able to.